src/APIFunctions/User.js

```diff
@@ -151,3 +152,35 @@ export async function checkIfUserExists(email) {
   });
   return status;
 }
+
+/**
+ * This function takes in a list of current users and returns a
+ * filtered user list that is determined by the filter id.
+ * @param {array} users array of all registered users
+ * @param {integer} filterID represents what to filter email by
+ * @returns {array} filtered array of users
+ */
+export function filterUsers(users, filterID) {
+  let filteredUsers = users.filter((user) => {
```

Comment:
`array.filter` will automatically omit elements from an array by us returning a boolean instead of `user` or `''`, read more [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter)